README.md

```diff
@@ -52,7 +52,7 @@ ZOSLIB is supported on the following hardware:
 * CMake 3.24.2-zos+
 * GNU Make 4.3+
 * IBM XL C/C++ V2.3.1 for z/OS V2.3 web deliverable (xlclang/xlcang++) or
-  IBM Open XL C/C++ 2.0 (clang/clang++)
+  IBM C/C++ for Open Enterprise Languages on z/OS 2.0.0 (clang/clang++)
```

Comment:
Shouldn't both be listed, and with XL stating (ibm-clang/ibm-clang++)?

Reply:
I added Open XL v2.1 too.  One thing I did notice is that the -m64 option is not being passed to the link step.  That's a problem in the CMakefile.txt.  We'll need to fix that.  For now people can just use ibm-clang64 as the command.